guides/common/modules/proc_importing-ansible-variables-from-yaml.adoc

```diff
@@ -0,0 +1,16 @@
+[id="importing-Ansible-variables-from-yaml-{context}"]
+= Importing Ansible variables from YAML files
```

Comment:
title does not match anchor and file name.

Reply:
file name does not match title+anchor. You'll also have to update the file name in guides/common/assembly_getting-started-with-ansible.adoc

Reply:
I think now I got it...